packages/react-composites/src/composites/CallComposite/styles/Call.styles.ts

```diff
@@ -8,5 +8,6 @@ export const callContainer = mergeStyles({
   width: '100%',
   overflow: 'auto',
   minHeight: '30rem',
-  minWidth: '50rem'
+  minWidth: '50rem',
+  boxShadow: 'rgba(0, 0, 0, 0.133) 0px 1.6px 3.6px 0px, rgba(0, 0, 0, 0.11) 0px 0.3px 0.9px 0px'
```

Comment:
might not be able to, but any chance this can make use of the theme box shadow value? https://developer.microsoft.com/en-us/fluentui#/styles/web/elevation

Reply:
Done. Thanks for calling this out. We should be respecting as much of the theming as possible in all our hand-rolled containers for Fluent components.